source/stylesheets/_tables.scss

```diff
@@ -0,0 +1,40 @@
+$table-border-color: #ddd;
+$table-zebra-color: #fafafa;
+
+.markdown-document table {
+  padding: 0;
+
+  tr {
+    border-top: 1px solid $table-border-color;
+    background-color: white;
+    margin: 0;
+    padding: 0;
+
+    &:nth-child(2n) {
+      background-color: $table-zebra-color;
+    }
+  }
+
+  tr th {
+    font-weight: bold;
+    border: 1px solid $table-border-color;
+    text-align: left;
+    margin: 0;
+    padding: 6px 13px;
+  }
+
+  tr td {
+    border: 1px solid $table-border-color;
```

Comment:
Thanks for this PR @sandersonet! I think this CSS rule is inadvertently being applied to code fences, too, since they're wrapped in `div.highlight table`. Not sure the best way to fix this, but see e.g., https://support.aptible.com/quickstart/ruby/rails/